config/production.json

```
@@ -116,8 +116,8 @@
 		"me/account/color-scheme-picker": true,
 		"me/vat-details": true,
 		"migration-flow/new-migration-instructions-step": false,
+    "migration-flow/introductory-offer": false,
```

Nit: it looks like the formatting is off here.
```suggestion
		"migration-flow/introductory-offer": false,
```